### ISSUE TITLE
NoReverseMatch on forms with ajax

### ISSUE THREAD

**Issue by lsetiawan** [ODM2/CZ-Manager#98]
I'm getting NoReverseMatch errors on change form with ajax.

For Example:
NoReverseMatch at /admin/odm2admin/units/2/change/

Reverse for 'odm2admin_cvunitstype_add' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []

**Comment by miguelcleon** [comment]
I'm not seeing this problem on the development or production server. I had seen this problem on the development server when I was using the python environment generated from the requirements.txt file. Maybe your seeing this on a local copy of ODM2 Admin?

**Comment by lsetiawan** [comment]
@miguelcleon Yes, I'm seeing this while I'm developing. Thanks. I will try to use an environment generated not from requirements.txt

**Comment by miguelcleon** [comment]
@lsetiawan ok, if we can figure out what is going wrong with that we should be able to help @ocefpaf  with the packaging. Is some requirement missing? I've been meaning to look into this more.
